src/js/components/Overview/charts/BentoPie.js

```diff
@@ -43,6 +43,8 @@ function BentoPie({ data, height }) {
     .filter((e) => !(e.x === 'missing'))
     .map((d) => ({ ...d, x: t(d.x) })); // Translating the labels
 
+  sort && data.sort((a, b) => a.y - b.y);
```

Comment:
i'd prefer if this were expanded out into an if statement despite this being more compact, since it's a mutating statement and not just a shortcut for rendering

Reply:
done